app/main.py

```diff
@@ -1,3 +1,5 @@
+from app.battle.battle import battle
+
 KNIGHTS = {
```

Comment:
mode dict with data to separate models

Reply:
you can be more precise, I don't quite understand what you're mean